packages/@aws-cdk/aws-apigatewayv2-integrations/lib/http/aws/integration.ts

```diff
@@ -0,0 +1,23 @@
+import {
```

Comment:
Can we move this here - https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/aws-apigatewayv2-integrations/lib/http/private/integration.ts ?

The 'private' directory here does not refer to private integration but rather classes that are private (should not be exported) to this module.